### PR TITLE
chore(bigtable): release v1.5.0

### PR DESCRIPTION
**Pull request by tritone**

### CODE REVIEW

**Comment by tritone** [comment]
FYI @liubonan @kolea2

File: bigtable/CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changes
 
+## v1.5.0
+- Add support for managed backups.
```
Comment (tbpg) [inline bigtable/CHANGES.md:4]: I'm surprised this is the only change. `release-please` and Conventional Commits will be really nice when it's all working.

Comment (tritone) [reply bigtable/CHANGES.md:4]: Yep-- previous release was < 1 month ago, and the only other changes were dependency updates :)

**Comment by kolea2** [comment]
Thank you!

**Pull request merged**